app/views/home/_constituency_form.html.haml

```diff
@@ -0,0 +1,14 @@
+= form_with url: pre_login_path, id: "login-form", local: true do |f|
+
+
+  %p.choose-constituency
```

Comment:
I'm not sure you want to use paragraphs for this, instead I would use divs with standard padding so each div gets its own padding. At the moment it isn't quite right, as the postcode field doesn't get the right padding.